.env.template

```
@@ -7,3 +7,7 @@ FLASK_DEBUG=true
 
 # Required to use Flask's session feature (for storing todos in a browser cookie in the first exercise): https://flask.palletsprojects.com/en/2.3.x/config/#SECRET_KEY
 SECRET_KEY=secret-key
+
+#add your Trello API Credentials here
+TRELLO_API_KEY = <FILL ME IN>
+TRELLO_API_TOKEN = <FILL ME IN>
```

Thanks for putting these here, I think there's a few more that you added to your code as you went though that would be good to document!

```suggestion
TRELLO_API_KEY = <FILL ME IN>
TRELLO_API_TOKEN = <FILL ME IN>

TRELLO_TODO_ID=<FILL ME IN>
TRELLO_DONE_ID=<FILL ME IN>
TRELLO_BOARD_ID=<FILL ME IN>
```